.github/workflows/unit_test_200gb_CI.yml

```diff
@@ -69,6 +69,14 @@ jobs:
           repository: marqo-ai/marqo-base
           path: marqo-base
 
+      # TODO Bake these into the AMIs later
```

Comment:
we removed this step when we move all the tests that require languagebind model to large model test. Are we moving any of those test back?

Reply:
I am not adding tests that requires model loading back. But I want to ensure ffmpeg can still work in the cpu environment.